src/algorithms/pagerank.c

```diff
@@ -0,0 +1,224 @@
+#include "pagerank.h"
+#include "../util/rmalloc.h"
+#include <assert.h>
+
+//------------------------------------------------------------------------------
+// scalar operators
+//------------------------------------------------------------------------------
+
+#define DAMPING 0.85
+
+void fdiff(void *z, const void *x, const void *y) {
```

Comment:
Maybe should be fDiffSquare?

Reply:
Not going to mess with LAGraph implementation in such away which will introduce a diff/delta which is absolutely not necessary